.circleci/config.yml

```diff
@@ -58,14 +58,14 @@ jobs:
       - run:
           name: Run UI tests against 2020.1 (earliest supported)
           command: ./scripts/run-ui-tests 2020.1
-      # We can't just store the results once at the end since they get overwritten with each UI test run.
-      - store_test_results:
-          path: uiTests/build/test-results/
       - run:
-          name: Run UI tests against 2020.1.2 (latest non-EAP & our build target)
+          name: Run UI tests against 2020.1.2
           command: ./scripts/run-ui-tests 2020.1.2
+      - run:
+          name: Run UI tests against 2020.2 (latest non-EAP & our build target)
+          command: ./scripts/run-ui-tests 2020.2
       - store_test_results:
-          path: uiTests/build/test-results/
+          path: uiTests/build/ui-test-results/
```

Comment:
And now it's sth like:
![image](https://user-images.githubusercontent.com/3383210/88852239-4ba8d980-d1ee-11ea-8ce3-99701786b853.png)